src/ol/layers/MapboxLayer.js

```diff
@@ -268,6 +315,9 @@ export default class MapboxLayer extends Layer {
       }
     }
     this.mbMap.on('idle', this.updateAttribution);
+    // returns value to allow testing easily.
+    // eslint-disable-next-line consistent-return
+    return style;
```

Comment:
Remove this.  Test the content of this.mbMap . mapboxgl.Map is already   mocked somewhere.